tools/regression_test.sh

```diff
@@ -313,7 +313,7 @@ function run_db_bench {
 }
 
 function set_async_io_parameters {
-  options=" --duration=800"
+  options=" --duration=500"
```

Comment:
Sorry I am bit confused here: why do we set a number less than 700 while the PR summary mentioned " Internally, the benchmark takes around 700 seconds"

Reply:
The thread takes around 700s irrespective of duration. However after 700 it checks if duration is still left to run and it will go for the next run (again 700s) making it total 1400s. After 1400s, it exits because 800 < 1400s.

Reply:
With 500, it won't go for next run because 500 < 700 and it will exit.